Jenkinsfile

```diff
@@ -921,6 +929,11 @@ void mavenNonDefaultBuild(BuildEnvironment buildEnv, String args, List<String> a
 					--fail-at-end \
 					$argsWithProjectSelection \
 	"""
+
+	if (incrementalBuild && 0 == sh(script: "test ! -s '${incrementalProjectsListFile}'", returnStatus: true)) {
```

Comment:
maybe we could do that before running the tests ^ . Or we' want the output of this exact build to determine if the stage needs to be marked as skipped?

Reply:
Well the file is created by gib during the Maven build, and we're only interested in it if it's empty, i.e. if the Maven build did close to nothing... so I don't think it's worth a separate Maven execution?

Reply:
thanks 😃 
yeah it seemed confusing, running the build to mark it skipped, but yeah it makes sense now 😃, we basically want  to see that nothing was tested on CI side without looking into the logs and so on, right ?

Reply:
Right:

```
	// In incremental builds, the Maven execution above
	// created a file listing projects relevant to the incremental build.
	// If it is empty, it means the incremental build didn't actually do anything,
	// so make sure to mark the stage as skipped.
```